src/main/features/directions-questionnaire/routes/hearing-exceptional-circumstances.ts

```diff
@@ -34,21 +41,42 @@ export default express.Router()
     FormValidator.requestHandler(ExceptionalCircumstances, ExceptionalCircumstances.fromObject),
     ErrorHandling.apply(async (req: express.Request, res: express.Response) => {
       const form: Form<ExceptionalCircumstances> = req.body
+      const party: MadeBy = getUsersRole(res.locals.claim, res.locals.user)
 
       if (form.hasErrors()) {
         renderPage(res, form)
       } else {
         const draft: Draft<DirectionsQuestionnaireDraft> = res.locals.draft
         const user: User = res.locals.user
+        if (party === MadeBy.CLAIMANT && res.locals.claim.response.directionsQuestionnaire) {
+          const defendantDirectionsQuestionnaire: DirectionsQuestionnaire = res.locals.claim.response.directionsQuestionnaire
 
+          if (form.model.exceptionalCircumstances.option === YesNoOption.YES.option) {
+            draft.document.hearingLocation.courtName = defendantDirectionsQuestionnaire.hearingLocation.courtName
+            form.model.reason = undefined
+          }
+        } else {
+          if (form.model.exceptionalCircumstances.option === YesNoOption.NO.option) {
```

Comment:
can you please make this an else if? no need to have the if inside the else if there's no other options

Reply:
done